app/src/main/java/com/nexenio/bleindoorpositioningdemo/ui/beaconview/map/BeaconMap.java

```diff
@@ -107,6 +121,15 @@ protected void drawBeacons(Canvas canvas) {
         for (Beacon beacon : beacons) {
             drawBeaconForeground(canvas, beacon, beaconCenterMap.get(beacon));
         }
+        // draw all distances
+        for (Beacon beacon : beacons) {
+            PointF beaconCenter = beaconCenterMap.get(beacon);
```

Comment:
This will result in the distance being drawn over the beacon. Always draw the background first.